### PR TITLE
Make body Streamable

### LINKED ISSUE #60: Find a solution for steaming responses
Currently, the only way to stream a response is to:

```php
$result = ...;
$info = $result->getInfo();
foreach($httpClient->stream($info['response']) as $chunk) {
  // ...
}
```

Do we want to provide a better way?

### PR DESCRIPTION
**Pull request by Nyholm**
I know I said we should wait with #60. But I couldn't stop thinking about it. 

This will fix #60

### CODE REVIEW

File: src/S3/S3Client.php [hunk 1]
```diff
@@ -237,46 +237,6 @@ public function copyObject($input = []): CopyObjectOutput
...
-     *   ResponseExpires?: \DateTimeImmutable|string,
-     *   VersionId?: string,
-     *   SSECustomerAlgorithm?: string,
-     *   SSECustomerKey?: string,
-     *   SSECustomerKeyMD5?: string,
-     *   RequestPayer?: string,
-     *   PartNumber?: int,
-     * }|GetObjectRequest $input
-     */
-    public function getObject($input = []): GetObjectOutput
```
Comment (jderusse) [inline src/S3/S3Client.php]: Is their a way to tell avoid such diff?
- tell to `nette/php-generator` to sort methods?
- add a php-cs-fixer on `/*/*Client` to sort methods?

**Comment by Nyholm** [comment]
This PR is wip

File: src/S3/S3Client.php (hunk 1 above)
Comment (Nyholm) [reply src/S3/S3Client.php]: > tell to nette/php-generator to sort methods?

No, and there is a massive "copy code" to achieve that. 

> add a php-cs-fixer on /*/*Client to sort methods?

I dont know. Can php-cs-fixer sort? Why not sort everything then?

**Comment by Nyholm** [comment]
This PR is ready for review

**Pull request merged**